tests/scripts/deploy_admission_controller.sh

```diff
@@ -11,16 +11,45 @@ BASE_DIR=$(cd "$(dirname "$0")"; pwd)
 WEBHOOK_CONFIG_NAME="rook-ceph-webhook"
 SERVICE_NAME="rook-ceph-admission-controller"
 
-INSTALL_SELF_SIGNED_CERT=true
 echo "$BASE_DIR"
 
-if [ "${INSTALL_SELF_SIGNED_CERT}" == true ]; then
-	"${BASE_DIR}"/webhook-create-signed-cert.sh --namespace ${NAMESPACE}
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
+timeout 150 sh -c 'until [ $(kubectl -n cert-manager get pods --field-selector=status.phase=Running|grep -c ^cert-) -eq 3 ]; do sleep 1; done'
+if [ $? -ne 0 ]; then
+  exit
 fi
+
 echo "Deploying webhook config"
+sleep 5
```

Comment:
Why this extra sleep? It's better to wait for a resource to be ready and retry.

Reply:
sometime pod are in running `STATUS` with `0/1` READY so it takes time to `1/1` READY(I faced this one when my system was very very slow ).

Reply:
So let's wait for `1/1`